src/components/graphQLConnect.ts

```diff
@@ -49,6 +49,12 @@ class GraphQLdb {
           isLastLevel
           isLastStage
           status
+          reviewer {
+            username?: string
+            firstName?: string
+            lastName?: string
+            email?: string
+          }
```

Comment:
```suggestion
          reviewer {
            id
            username
            firstName
            lastName
            email
          }
```
It's a query, not a type! 😆

Reply:
Hhahaha bad copy! :P